src/components/post-comments.jsx

```diff
@@ -57,8 +57,13 @@ export default (props) => {
       props.toggleCommenting(props.post.id);
     }
 
-    const updatedCommentText = `@${username} ` + (props.post.newCommentText || '');
-    props.updateCommentingText(props.post.id, updatedCommentText);
+    const text = (props.post.newCommentText || '');
+    const check = `@${username}$`;
+
+    if (!text.match(check)) {
```

Comment:
Does it work correctly? I thought `match()` takes a _regexp_ as a parameter, not a _string_. Or am I missing something?